### PR TITLE
Closes #42 remove commented code

### LINKED ISSUE #42: Cleanup commented code
In order to keep a clean codebase, we should clean up any code that is not in use. This includes code that has been commented out, without a 'TODO' comment.

# Goal
Keep a tidy codebase by removing commented-out code.

# Task
Clean up the following files/lines that have commented out code.

## /api/models/civi.py
- [x] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/models/civi.py#L99-L100

## /api/models/hashtag.py
- [x] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/models/fact.py#L7

## /api/models/rebuttal.py
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/models/rebuttal.py#L4
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/models/rebuttal.py#L13

## /api/models/representative.py
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/models/representative.py#L10

## /api/models/response.py
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/models/response.py#L4
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/models/response.py#L14

## /api/read.py
- [x] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/read.py#L14-L48
- [x] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/read.py#L76-L79
- [x] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/read.py#L140
- [x] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/read.py#L158-L159
- [x] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/read.py#L162-L165
- [x] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/read.py#L177-L179
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/read.py#L188-L212
- [x] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/read.py#L226-L228

## /api/urls.py
- [x] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/urls.py#L6-L10
- [x] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/urls.py#L27-L29
- [x] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/urls.py#L41-L47

## /api/write.py
- [x] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/write.py#L10
- [x] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/write.py#L46-L102
- [x] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/write.py#L126-L135
- [x] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/write.py#L145-L146
- [x] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/write.py#L168-L170
- [x] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/write.py#L555-L581
- [x] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/write.py#L735-L737

## /frontend_views/views.py
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/frontend_views/views.py#L95
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/frontend_views/views.py#L100-L106

### PR DESCRIPTION
**Pull request by tractaylor**
Finished removing the commented out code listed in this issue.

Looking at the files listed below, I am not sure why https://github.com/CiviWiki/OpenCiviWiki/commit/bc7d3784939129a8b67dbf1f0ef73b71430453a8 is showing up in this pull request, I believe its already in a pull request. Same goes for all others except for the last 3 commits that reference issue 42... 

Did I do something incorrect?

### CODE REVIEW

File: project/api/models/civi.py [hunk 1]
```diff
@@ -131,7 +128,7 @@ def score(self, request_acct_id=None):
         D = -1
         A = 1
         SA = 2
-        owner_id = self.author
+        #owner_id = self.author
```
Comment (brylie) [inline project/api/models/civi.py]: Remove this commented code, either in this PR or #39.

File: project/api/write.py [hunk 2]
```diff
@@ -462,7 +386,7 @@ def clearProfileImage(request):
             account.save()
 
             return HttpResponse('Image Deleted')
-        except Exception as e:
+        except Exception:
```
Comment (brylie) [inline project/api/write.py]: Removing the variable `e` from this line may cause line **478** to fail. Rather, rename `e` to be `exception`, and update line **478** to use the `exception` variable.

Comment (tractaylor) [reply project/api/write.py]: I don;t understand how the except at line 465 can affect anything after the line 466 since its returning.  Line 478 is a new try block?  Unless you mean to update line 466 to use the exception object instead of using return HttpResponseServerError(reason=str(default)).  So I could change it to be return HttpResponseServerError(reason=str(e)).

Comment (brylie) [reply project/api/write.py]: Ah, OK. Nevermind. The code truncation makes it hard to follow the flow of the files sometimes.

**Comment by tractaylor** [comment]
This should be ready to review again.

**Comment by mdwest** [comment]
Is this ready to be merged?

**Comment by tractaylor** [comment]
I think it was just waiting for another review after I made the changes requested by @brylie

**Comment by mdwest** [comment]
Great! @brylie can you confirm that and merge this pull request!

**Comment by brylie** [comment]
@tractaylor please resolve conflicts, so we can merge.

**Comment by brylie** [comment]
@tractaylor still waiting for you to resolve conflicts in two files:

- project/api/read.py
- project/api/write.py

Let me know if you are having difficulties resolving these conflicts.

**Comment by tractaylor** [comment]
@brylie Hi, yes I am having difficulty fixing the conflicts.  I cannot get the latest dev branch to sync into my branch to so I can fix the lines.  Looks like the issue was caused from another PR was merged that had changes around removing some commented out code.  Do you know the steps I need to do?  I have tried, while in my branch to git rebase --dev but that didnt  help

**Comment by brylie** [comment]
@tractaylor just use the GitHub 'Resolve conflicts' UI:

https://github.com/CiviWiki/OpenCiviWiki/pull/72/conflicts

**Comment by tractaylor** [comment]
The github UI doesn't allow a merge.  That is my problem.  I tried to get the conflict on my dev machine but cannot seem to get the state.

**Comment by brylie** [comment]
The Github UI will allow you to resolve the conflicts. You don't need to merge the PR. Just resolve the conflicts by editing the files:

https://github.com/CiviWiki/OpenCiviWiki/pull/72/conflicts

**Comment by tractaylor** [comment]
@brylie I had looked at the conflicts in the UI for many days and never knew I could edit them.  I had been trying to get the conflicts on my dev machine for many days.  Thanks for telling that.  I had totally missed it
The conflicts should be resolved now.
Sorry about that.

**Comment by brylie** [comment]
I cleaned up the last commented line with https://github.com/CiviWiki/OpenCiviWiki/pull/72/commits/103340370c505ac50ea74af2caf48f4337520022 and will merge this now.
